numba_dpex/core/utils/kernel_launcher.py

```diff
@@ -413,10 +450,73 @@ def set_kernel(self, sycl_kernel_ref: llvmir.Instruction):
         """Sets kernel to the argument list."""
         self.arguments.sycl_kernel_ref = sycl_kernel_ref
 
+    def set_kernel_from_spirv(self, kernel_module: SPIRVKernelModule):
+        """Returns the pointer to the sycl::kernel object in a passed in
```

Comment:
The docstring is not correct. You are not passing in a `sycl::kernel_bundle` object.